src/arche/rules/category.py

```diff
@@ -14,31 +14,33 @@ def get_difference(
     result = Result("Category Coverage Difference")
 
     for c in category_names:
-        cats = pd.DataFrame(
-            {
-                "s": source_df[c].value_counts(normalize=True, dropna=False),
-                "t": target_df[c].value_counts(normalize=True, dropna=False),
-            }
-        ).fillna(0)
-        cat_difs = (
-            ((cats["s"] - cats["t"]) * 100)
-            .abs()
-            .sort_index(kind="mergesort")
-            .sort_values()
-            .round(decimals=2)
+        cats = (
```

Comment:
I did some experiments with 150K and this could be really slow operation for columns with many categories.

```
df = pd.read_json("shared/qa/Experiments/data/items_epicsports_150k_nested.json")
s = df.variant 
len(s.value_counts(dropna=False, normalize=True))
```

Reply:
What you suggest?